lib/msal-common/src/request/broker/BrokeredSilentFlowRequest.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SilentFlowRequest } from "../SilentFlowRequest";
+
+/**
+ * SilentFlow parameters passed by the user to retrieve credentials silently
+ * - scopes                 - Array of scopes the application is requesting access to.
```

Comment:
We should look at these comments, I'm not a fan of the typedocs they generate, I think theres a better way. Plus, it is duplicitive to document `SilentFlowRequest` params here.

Reply:
Happy to discuss ways to improve the typedoc comments